src/store/user.js

```diff
@@ -30,6 +30,7 @@ const actions = {
     const logoutFinalizier = (forceRedirect = false) => {
```

Comment:
While you're at it - could you fix the typo in `logoutFinalizer` pretty please? :-)

Reply:
done ;)